src/scope/core/pipelines/krea_realtime_video/modules/causal_model.py

```diff
@@ -512,7 +512,11 @@ def qkv_fn(x):
             if kv_cache_attention_bias != KV_CACHE_ATTENTION_BIAS_DISABLED:
                 # Use flex_attention with bias to mitigate error accumulation in past frames
                 # log_scale in (0, 1]: smaller values = less attention to past frame tokens
-                log_scale = math.log(kv_cache_attention_bias)
+                log_scale = torch.tensor(
```

Comment:
Just to clarify - were these changes to fix any behavior or just for consistency to always use tensors?